exporter/azureblobexporter/jsonl_utils.go

```diff
@@ -0,0 +1,23 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package azureblobexporter
+
+import (
+	"bytes"
+	"regexp"
+)
+
+func removeNullValues(data []byte) []byte {
+	if !bytes.Contains(data, []byte(": null")) {
+		return data
+	}
+	
+	nullFieldRegex := regexp.MustCompile(`"[^"]+"\s*:\s*null\s*,?\s*`)
```

Comment:
Compiling regexp each time is an expensive operation, please consider pre computing this or solving it without regexp.